IMAGES.rst

```diff
@@ -39,19 +39,19 @@ The images are named as follows:
 
 where:
 
-* ``BRANCH_OR_TAG`` - branch or tag used when creating the image. Examples: ``master``, ``v1-10-test``, ``1.10.14``
-  The ``master`` and ``v1-10-test`` labels are built from branches so they change over time. The ``1.10.*`` and in
-  the future ``2.*`` labels are build from git tags and they are "fixed" once built.
-* ``PYTHON_MAJOR_MINOR_VERSION`` - version of python used to build the image. Examples: ``3.5``, ``3.7``
+* ``BRANCH_OR_TAG`` - branch or tag used when creating the image. Examples: ``master``,
+  ``v2-0-test``, ``v1-10-test``, ``2.0.0``. The ``master``, ``v1-10-test`` ``v2-0-test`` labels are
+  built from branches so they change over time. The ``1.10.*`` and ``2.*`` labels are built from git tags
+  and they are "fixed" once built.
+* ``PYTHON_MAJOR_MINOR_VERSION`` - version of python used to build the image. Examples: ``3.6``, ``3.7``
 * The ``-ci`` suffix is added for CI images
-* The ``-manifest`` is added for manifest images (see below for explanation of manifest images)
```

Comment:
Why this line is removed?

Reply:
IF it's removed because it's no longer applicable, section `Image manifests` should be removed as well?

Reply:
Good point. Mistake. We still have the manifest image but it contains a bit different data (randomly generated UUID at the point of building image that is a decision point on whether it is (way) faster to pull + build or just to build the image. I updated the description of the manifest image and restored the deleted line here.